Makefile

```diff
@@ -317,4 +330,22 @@ clean:
 	-rm -rf $(PWD)/bin
 	-rm -rf cover.out
 	-rm -rf coverprofile.out
-
+	-rm -f ecs-agent.spec
+	-rm -f ecs.service
+	-rm -f amazon-ecs-volume-plugin.service
+	-rm -f amazon-ecs-volume-plugin.socket
+	-rm -f ./sources.tgz
+	-rm -f ./amazon-ecs-agent
+	-rm -f ./amazon-ecs-agent-*.rpm
+	-rm -f ./ecs-agent-*.tar
+	-rm -f ./ecs-agent-*.src.rpm
+	-rm -rf ./ecs-agent-*
+	-rm -rf ./BUILDROOT BUILD RPMS SRPMS SOURCES SPECS
+	-rm -rf ./x86_64
+	-rm -f ./amazon-ecs-agent_${VERSION}*
+	-rm -f .srpm-done .rpm-done .generic-rpm-done
+	-rm -f .deb-done
+	-rm -f amazon-ecs-cni-plugins.tar.gz
+	-rm -f amazon-vpc-cni-plugins.tar.gz
+	-rm -rf ../amazon-ecs-cni-plugins
```

Comment:
why is it required to remove package as well?

Reply:
I just went ahead and removed all the packages that are brought in or will be brought in as more rpm targets are made. This way, the agent files can be cleaned with just running `make clean`